README.md

```diff
@@ -127,127 +127,175 @@ future state of the libraries and to discover projects for contribution.
+have an initial version submitted as a pull request, it is time to reach out to
+the community and any relevant stakeholders and solicit their feedback.
+
+Use the [cdk.dev](https://cdk.dev) Slack workspace, Twitter and any other
+relevant forum to publish your RFC and ask the community to provide their
+feedback.
+
+### 6. Iterate on the RFC document
+
+This is where the fun begins. Once you start receiving feedback on your pull request,
+iterate on your document. Take time to read the comments, understand where people are coming
+from, respond politely and iterate.
+
+A few tips:
+
+- If you decide to resolve a comment without addressing it, take the time to explain.
+- Try to understand where people are coming from. If a comment seems off, ask folks to elaborate and
+  describe their use case or provide concrete examples.
+- Work with your approver(s): if there are disagreements, @mention them in a comment and ask them
+  to provide their opinion.
+- Be patient: it sometimes takes time for an RFC to converge. Our experience shows that some ideas
+  need to "bake" and solutions oftentimes emerge via a healthy debate. We've had RFCs that took
+  months to resolve.
+- Not everything must be resolved in the first revision. It is okay to leave
+  some things to resolve later. Make sure to capture them clearly and have an
+  agreement about that. We oftentimes update an RFC doc a few times during the
+  implementation.
+
+### 7. Final Comments Period
+
+At some point, you've reached consensus about most issues that were brought up
+during the review period. In consultation with the approvers, a driver can
+announce that the RFC enters "final comments period", which means that within a
+~week, if no major concerns are raised, the RFC will be approved and merged.
+
+Add a comment on the RFC pull request and tracking issue that the RFC entered
+this stage so that all relevant stakeholders will be notified.
+
+### 8. Obtain final approval
+
+Once the final comments period is over, and the author has made final edits, the
+RFC can be approved by the approvers and merged into the repository.
+
+NOTE: only RFCs that were approved by approvers should be merged.
+
+### 9. Implementation
+
+For large changes, we highly recommend creating an implementation plan which
+lists all the tasks required. In many cases, large implementation  should be
+broken down and released via multiple iterations. Devising a concrete plan to
+break down the break can be very helpful.
+
+The implementation plan should be submitted through a PR that adds an addendum
+to the RFC document and seeks the approval of any relevant stakeholders.
+
+Throughout this process, update the tracking issue:
+
+- Add the alias of the "implementation lead"
+- Execution plan submitted (label: `status/planning`)
+- Plan approved and merged (label: `status/implementing`)
+- Implementation complete (label: `status/done`)
+
+## State Diagram
```

Comment:
This section needs updating as well.

Reply:
Not sure - it's generally the same state flow